src/features/ProfilePage/components/index.ts

```diff
@@ -0,0 +1,20 @@
+import styled from 'styled-components';
+import { palette } from '@/components/constants';
+import Text from '@/components/Text';
+
+export const Section = styled.div`
+  border-bottom: 1px solid ${palette.blueDark};
+  display: flex;
+  flex-direction: column;
+  padding: 1rem 0;
+`;
+
+export const ButtonRow = styled.div`
+  display: flex;
+  gap: 2rem;
+  justify-content: center;
+`;
+
+export const Value = styled(Text)`
+  margin: 0.5rem 0 0 0;
+`;
```

Comment:
I think this pattern could be used in other parts of our codebase too. 
I dont like the idea of creating a component-file for every single styled-wrapper and when there is a lot of styled-divs or so in one component it also seems a bit silly. And we most likely have a lot of duplicate/similar containers in the codebase atm.

Reply:
Yup this could improve consistency!